src/Table.php

```diff
@@ -426,6 +359,43 @@ public function renderView()
         return View::renderView();
     }
 
+    public function renderRow()
+    {
+        $this->t_row->set($this->model);
+
+        if ($this->use_html_tags) {
+            // Prepare row-specific HTML tags.
+            $html_tags = [];
+
+            foreach ($this->hook('getHTMLTags', [$this->model]) as $ret) {
+                if (is_array($ret)) {
+                    $html_tags = array_merge($html_tags, $ret);
+                }
+            }
+
+            foreach ($this->columns as $name => $columns) {
+                if (!is_array($columns)) {
+                    $columns = [$columns];
+                }
+                $field = $this->model->hasElement($name);
+                foreach ($columns as $column) {
+                    if (!method_exists($column, 'getHTMLTags')) {
```

Comment:
please simplify this to:
```
if (method_exists($column, 'getHTMLTags')) {
    $html_tags = array_merge($column->getHTMLTags($this->model, $field), $html_tags);
}
```

No need for negation and `continue;` here.

Reply:
i dont understand, please propose through a separate PR.